client/src/context/GameContext.js

```diff
@@ -35,6 +35,11 @@ const GameContextProvider = ({ children }) => {
         msg: `${joinedPlayer} joined the game!`,
       });
     });
+
+    sockets.on("game-started", () => {
+      setGame((game) => ({ ...game, isStarted: true }));
+    });
+
```

Comment:
These game-start parts will be modified when integrating with game-integration branch. After integrating between RTC and game, you need to check it.